src/Config.php

```diff
@@ -46,7 +48,11 @@ public function config(): PhpCsFixerConfig
             $this->getAllFiles($files, $directory);
         }
 
-        $finder = Finder::create()->directories()->append($files);
+        $filteredFiles = $this->ignoreMarkedFiles
+            ? array_filter($files, fn(string $file): bool => !str_contains(file_get_contents($file), self::IGNORE_TAG))
+            : $files;
+
+        $finder = Finder::create()->directories()->append($filteredFiles);
```

Comment:
Maybe this would be better approach? https://symfony.com/doc/current/components/finder.html#file-contents

Reply:
I tried this but not works